### PR TITLE
[DS-293]: [Pyxis Elm] Implement Counter component

### PR DESCRIPTION
**Pull request by brianvanburken**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/DS-293

## 🚀 New Features
Implementation of Counter component in pyxis-elm.
## 📝 Docs
The documentation of the Counter has been updated.

### CODE REVIEW

File: packages/pyxis-elm/src/Pyxis/Components/Field/Counter.elm [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+
+
+## Rendering
+
+@docs render
+
+-}
+
+import Html exposing (Html)
+import Html.Attributes as Attr
```
Comment (emanuelemacri) [inline packages/pyxis-elm/src/Pyxis/Components/Field/Counter.elm:72]: we generally use it like this
```suggestion
import Html.Attributes
```

File: packages/pyxis-elm/src/Pyxis/Components/Field/Counter.elm [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+        , maxValue : Int
+        , step : Int
+        , onChange : Maybe msg
+        }
+
+
+{-| Initialize the Counter internal state. This belongs to your app's `Model`.
+-}
+init : Int -> Int -> Int -> Model msg
+init initialValue minValue maxValue =
```
Comment (emanuelemacri) [inline packages/pyxis-elm/src/Pyxis/Components/Field/Counter.elm:200]: Usually our `init` is much simpler, but here we actually need a lot of values. @Gsciarra what do you say?
For better understanding we can create type aliases, how about that?
```suggestion
init : InitialValue -> MinValue -> MaxValue -> Model msg
init initialValue minValue maxValue =
```

File: packages/pyxis-elm/src/Pyxis/Components/Field/Counter.elm [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+        |> applyConfigIfJust Label.withSubText subLabel
+        |> Label.render
+
+
+{-| Internal.
+-}
+applyConfigIfJust : (a -> b -> b) -> Maybe a -> b -> b
+applyConfigIfJust f a =
+    Maybe.map f a
+        |> Maybe.withDefault identity
```
Comment (emanuelemacri) [inline packages/pyxis-elm/src/Pyxis/Components/Field/Counter.elm:355]: maybe this can be put in the commons?

**Review by emanuelemacri — CHANGES_REQUESTED** [review]
Some questions or small fixes, and I found a little visual issue:
The `.counter__content` class should have a `text-align: center;` properties.

File: packages/pyxis-elm/src/Pyxis/Components/Field/Counter.elm (hunk 2 above)
Comment (brianvanburken) [reply packages/pyxis-elm/src/Pyxis/Components/Field/Counter.elm:200]: Yeah, I agree. The order of the Int's without help from an IDE would be confusing.

**Review by emanuelemacri — APPROVED** [review]
LGTM

**Pull request merged**